SavedInstances/MythicPlus.lua

```diff
@@ -32,6 +32,9 @@ local KeystoneAbbrev = {
 }
 addon.KeystoneAbbrev = KeystoneAbbrev
 
+-- Frames
+local SavedInstancesKeyExport, SavedInstancesKeyExportScroll, SavedInstancesKeyExportScrollText

```

Comment:
Unused vars

Reply:
Here you can simply remove them, since these vars are not being used is anywhere in the file. (yes, not used by `CreateFrame` either)